autometa/common/external/hmmer.py

```diff
@@ -280,4 +259,25 @@ def main(args):
         '--parallel', help="Enable GNU parallel", action='store_true')
     parser.add_argument('--verbose', help="add verbosity", action='store_true')
     args = parser.parse_args()
-    main(args)
+
+    if args.verbose:
+        logger.setLevel(logger.DEBUG)
+    result = hmmscan(
+        orfs=args.orfs,
+        hmmdb=args.hmmdb,
+        outfpath=args.hmmscan,
+        cpus=args.cpus,
+        force=args.force,
+        parallel=args.parallel,
+        log=args.log)
+
+    result = filter_markers(
+        infpath=result,
+        outfpath=args.markers,
+        cutoffs=args.cutoffs,
+        orfs=args.orfs,
+        force=args.force)
```

Comment:
How do we account for when a user is supplying hmmscan results and just want to filter the results given the cutoffs? 

### Expected behavior:

Filter results given a previously generated hmmscan table

### Scenario

1. I have hmmscan results from the bacteria single-copy marker genes.
2. I do **not** want to overwrite these results.
3. I want to filter them based on the respective cutoffs.

### **Error encountered:** 
When I run `hmmer.py` I receive `FileExistsError` for my hmmscan results and the program exits. 
Can I filter these results without having to overwrite my previously generated hmmscan table?